Client/Frontend/Strings.swift

```diff
@@ -21,6 +21,7 @@ extension Strings {
     public static let CancelString = NSLocalizedString("Cancel", comment: "Label for Cancel button")
     public static let NotNowString = NSLocalizedString("Toasts.NotNow", value: "Not Now", comment: "label for Not Now button")
     public static let AppStoreString = NSLocalizedString("Toasts.OpenAppStore", value: "Open App Store", comment: "Open App Store button")
+    public static let PageActionMenuUndoAction = NSLocalizedString("Toasts.UndoRemoveFromBookmark", value: "Undo", comment: "Undo the action just performed in PageActionMenu")
```

Comment:
Let's just name this 'Undo' so that it is more general and can be used in other places if need be.

Reply:
I was wondering if this will be okay; `public static let Undo = NSLocalizedString("Toasts.GeneralUndo", value: "Undo", comment: "Undo the action just performed"` Or should it be "UndoString" to follow along with the lines just above it?

Reply:
Let's just do `public static let UndoString = NSLocalizedString("Toasts.Undo", value: "Undo", comment: "Label for button to undo the action just performed")`